_includes/footer.html

```diff
@@ -57,6 +57,7 @@ <h2 class="wb-inv">{{ site.data.i18n.general.footer.AboutSite[page.lang] }}</h2>
 <script src="../assets/js/src/more-group.js"></script>
 <script src="../assets/js/src/date.js"></script>
 <script src="../assets/js/src/slugify.js"></script>
+<script src="../assets/js/src/licences.js"></script>
```

Comment:
Not sure this needs to load on each page, but it looks like there are a bunch of files we're already doing that with